doc/extras/building_vtk.rst

```diff
@@ -32,19 +32,25 @@ wheel size.
 
     #!/bin/bash
 
-    # install build dependencies (Linux/Debian)
-    sudo apt-get install ninja-build cmake libgl1-mesa-dev python3-dev
+    # Install build dependencies
+
+    # Linux/Debian
+    sudo add-apt-repository -y ppa:deadsnakes/ppa  # if on Ubuntu 20.04 or 18.04, building 3.10 for example
```

Comment:
Do you think we should add some warnings/caveats about adding a PPA in our recommended workflow? E.g. https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa says

> Disclaimer: there's no guarantee of timely updates in case of security problems or other issues. If you want to use them in a security-or-otherwise-critical environment (say, on a production server), you do so at your own risk.

And adding a PPA is not a trivial matter (in terms of security etc., rather than actually doing it). There could even be technical issues e.g. https://stackoverflow.com/questions/66597894/why-cannot-add-ppa-deadsnakes.

Reply:
In practice I doubt these things will matter. For me adding a PPA has never really been problematic, and I just verified both of these worked (for 18.04 and 20.04). I guess we could add some warning, but these are all really just suggestions for people to take or leave anyway. But if you don't do this step, you can't build for 3.10 for example on either current LTS Ubuntu version, so it seems like a valuable suggestion

Reply:
We should definitely add this step, I'm only wondering about adding some explanation about it. But if you think it's "safe" I don't object, it's just something that came to mind :) Maintainers might chime in regarding this too.

Reply:
This is a well known repository. I see no issues adding it.